hw/top_earlgrey/data/ip/chip_aon_timer_testplan.hjson

```diff
@@ -3,137 +3,254 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name: chip_aon_timer
-  testpoints: [
+  testpoints:
+  [
     {
       name: chip_sw_aon_timer_wakeup_irq
-      desc: '''Verify the AON timer wake up interrupt in normal operating state.
+      desc:
+        '''
+        Verify the AON timer wake up interrupt in normal operating state.
 
-            - Program the PLIC to let the AON timer wake up interrupt the CPU.
-            - Program the AON timer to generate the wake up timeout interrupt after some time.
-            - Issue a WFI to wait for the interrupt to trigger.
-            - Service the interrupt when it triggers; verify that it came from AON timer.
-            - Verify that the interrupt triggered only after the timeout elapsed.
-            '''
+        - Program the PLIC to let the AON timer wake up interrupt the CPU.
+        - Program the AON timer to generate the wake up timeout interrupt after some time.
+        - Issue a WFI to wait for the interrupt to trigger.
+        - Service the interrupt when it triggers; verify that it came from AON timer.
+        - Verify that the interrupt triggered only after the timeout elapsed.
+        '''
       stage: V2
       si_stage: SV3
-      lc_states: ["DEV", "PROD", "RMA"]
-      features: ["AON_TIMER.WAKEUP.INTERRUPT"]
-      tests: ["chip_sw_aon_timer_irq"]
-      bazel: ["//sw/device/tests:aon_timer_irq_test"]
+      lc_states:
+      [
+        DEV
+        PROD
+        RMA
+      ]
+      features:
+      [
+        AON_TIMER.WAKEUP.INTERRUPT
+      ]
+      tests:
+      [
+        chip_sw_aon_timer_irq
+      ]
+      bazel:
+      [
+        "//sw/device/tests:aon_timer_irq_test"
+      ]
```

Comment:
I'm not sure I prefer this formatting

Reply:
I also find this formatting to be very verbose/hard to read, is there a way to control the formatter ?

Reply:
I also don't like it, and there's no way to control it. I'll have to write a simple serializer for the python `dict` I think. I'll investigate.

Reply:
I changed the formatting to be very similar to the one mostly used.